src/lib/operators/join_sort_merge.hpp

```diff
@@ -42,6 +42,11 @@ class JoinSortMerge : public AbstractJoinOperator {
     OutputWriting
   };
 
+  // Tasks are added to the scheduler in case the number of elements to process is above JOB_SPAWN_THRESHOLD. If not,
+  // the task is executed directly. This threshold needs to be re-evaluated over time to find the value which gives the
+  // best performance.
+  static constexpr auto JOB_SPAWN_THRESHOLD = 500;
```

Comment:
```
// Tasks are added to the scheduler in case the number of rows to process is above JOB_SPAWN_THRESHOLD. If not,
  // the task is executed directly. This threshold has been determined by executing a multi-threaded and shuffled TPC-H
  // run (28 cores and 50 clients). With larger system changes (e.g., scheduling), the threshold needs to be
  // re-evaluated again.
```

Ok?

Reply:
Yes. By the way: `troll` is the autocorrection's version of `trello`.

Reply:
Freudian auto correct.